modules/cardano-node.nix

```diff
@@ -142,7 +142,14 @@ in {
       "${toString hour} * * * * root /run/current-system/sw/bin/systemctl restart cardano-node"
     ];
 
-    networking.firewall.allowedTCPPorts = [ cfg.port ];
+    networking.firewall = {
+      enable = mkForce true;
```

Comment:
By default firewall is on. We should get rid of our `false` overrides in the codebase so that this line is redundant.